tests/unit/symbolic_regression/agraph/test_agraph_mutation.py

```diff
@@ -200,10 +321,172 @@ def test_mutate_variable(single_variable_agraph, sample_component_generator):
                               command_probability=0.0,
                               node_probability=0.0,
                               parameter_probability=1.0,
-                              prune_probability=0.0)
+                              prune_probability=0.0,
+                              fork_probability=0.0)
     child = mutation(single_variable_agraph)
     p_stack = single_variable_agraph.command_array
     c_stack = child.mutable_command_array
 
     assert p_stack[-1, 1] != c_stack[-1, 1]
     assert p_stack[-1, 2] != c_stack[-1, 2]
+
+
+def command_array_is_valid(individual):
+    for i, (command, op1, op2) in enumerate(individual.command_array):
+        if not IS_TERMINAL_MAP[command]:
+            if op1 >= i or op2 >= i:
+                return False
+    return True
+
+
+@pytest.mark.parametrize("repeats", range(5))
```

Comment:
These tests are nice, all pretty straightforward and easy to understand

Reply:
Thank you!